pypit/arutils.py

```diff
@@ -190,15 +192,17 @@ def find_peaks(yval, siglev=10.):
       Sigma level for detection
     """
     # Calculate RMS
-    yclipped = sig_clip(yval)
+    from pypit import arcyarc
+    yclipped = sigma_clip(yval)
     rms = np.std(yclipped)
     #
-    tpixt, num = arcyarc.detections_sigma(yval,np.array([rms]*mid_row.shape[0]),np.zeros(yval.shape[0],dtype=np.int),siglev/2.0,siglev) 
+    #tpixt, num = arcyarc.detections_sigma(yval,np.array([rms]*mid_row.shape[0]),np.zeros(yval.shape[0],dtype=np.int),siglev/2.0,siglev)
+    debugger.set_trace()  # FIX THE ROW ABOVE
```

Comment:
Should something be returned from this function after rms is calculated?  If there's a bug here, perhaps this should be filed as an issue ---  to remind me to look into it  :-)